docs/runtimes/mlrun_code_annotations.ipynb

```diff
@@ -0,0 +1,1253 @@
+{
```

Comment:
<p>Convert the function with <code>mlrun.code_to_function</code> and run the handler. Note the returned value under <code>results</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/1456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>